site/content/docs/4.3/content/typography.md

```diff
@@ -239,15 +239,15 @@ Remove the default `list-style` and left margin on list items (immediate childre
 </ul>
 {{< /example >}}
 
-### Inline
+### Inline lists
```

Comment:
By code, this works, but for the mental model of it, I wouldn't think to use grid classes to make something appear what was previously `display: inline-block`. Feels like this exchanges minor file size for a relationship we don't need (between quick text lists and gridded content).

Reply:
Ok, I just restored the inline lists.